dev/DropDownButton/DropDownButton.xaml

```diff
@@ -77,6 +77,9 @@
                                         <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="Foreground">
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonForegroundDisabled}" />
                                         </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ChevronTextBlock" Storyboard.TargetProperty="Foreground">
```

Comment:
We should probably just delete DropDownButton_rs1.xaml since RS1 is no longer supported.

Reply:
Should we do this in this PR or should we create a separate issue/PR where we get rid of all RS1 files?

Reply:
I was thinking DropDownButton only since you are already here. But yeah, I guess it's out of the scope of this PR, a separate issue is probably better.